server/api/users/index.get.ts

```diff
@@ -0,0 +1,27 @@
+import { UserDbClient } from "~/server/utils/database/user-db-client/user-db-client";
+import { ApiResponse } from "~/types/api-response/api-response";
+import { User } from "~/types/user/user";
+
+export default defineEventHandler<Promise<ApiResponse<User[]>>>(
+  async (event) => {
+    const dbClient = new UserDbClient();
+    const apiResponse = {} as ApiResponse<User[]>;
+
+    try {
+      const users = await dbClient.getAllUsers();
+      setResponseStatus(event, 201);
```

Comment:
Since no resource is being created, `200` would be best practice response. Or you could delete this statement as Nuxt will default to 200 anyway.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
```suggestion
      setResponseStatus(event, 200);
```

Reply:
Will remove this in the next commit